src/main/groovy/io/invertase/gradle/build/ProjectExtension.groovy

```diff
@@ -121,9 +121,9 @@ class ProjectExtension {
     }
 
     if (defaulted) {
-      println ":${project.name}:${scope}.${name} using default value: ${defaultValue}"
+      println ":${project.name}:${scope}.${name} using default value: ${value}"
     } else {
-      println ":${project.name}:${scope}.${name} using custom value: ${defaultValue}"
```

Comment:
This line specifically was the problem. It must be 'value' here. I changed both though just in case - the method returns 'value' so might as well always report what it returns?